libff/algebra/curves/tests/test_groups.cpp

```diff
@@ -412,3 +414,21 @@ TEST(TestGroups, BN128)
     test_mul_by_cofactor<G2<bn128_pp>>();
 }
 #endif
+
+#endif // #if 0 // BLS12_381 debug (VV)
+
+TEST(TestGroups, BLS12_381)
+{
+    bls12_381_pp::init_public_params();
+    test_group<G1<bls12_381_pp>>();
+    test_output<G1<bls12_381_pp>>();
+    test_group<G2<bls12_381_pp>>();
+    test_output<G2<bls12_381_pp>>();
+#if 0  // bls12-381: temporary disable for debug (VV)
```

Comment:
did you get to the root of the serialization issues?

Reply:
No. Still looking at it.

Reply:
Now fixed.